lib/ethers.ex

```diff
@@ -153,6 +153,23 @@ defmodule Ethers do
     |> post_process(nil, :get_transaction)
   end
 
+  @doc """
+  Returns the native transaction (ETH) receipt by transaction hash.
```

Comment:
What does `ETH` and native transaction mean in this context?

Can we just say `Returns receipt of a transaction by it's hash`?

Reply:
Yup, makes sense